### PR TITLE
Fix compare_tools.py

### PR DESCRIPTION
**Pull request by jbkoh**
- remove unnecessary imports
- new version style
- add README

### CODE REVIEW

File: tools/compare_versions/compare_versions.py [hunk 1]
```diff
@@ -6,18 +6,26 @@
 import argparse
 import semver
 
+import pdb
```
Comment (gtfierro) [inline tools/compare_versions/compare_versions.py]: Can you remove the `pdb` import?

**Comment by gtfierro** [comment]
We should probably also add `semver` to the `requirements.txt` for the Brick repo

File: tools/compare_versions/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Brick Version Comparison
+------------------------
+
+# How to use it?
+- `python tools/compare_versions/compare_versions.py --help` for getting help
+- `python --oldbrick 1.0.3 https://github.com/BrickSchema/Brick/releases/download/v1.0.3/Brick.ttl --newbrick 1.1.0 ./Brick.ttl
```
Comment (gtfierro) [inline tools/compare_versions/README.md]: Does this really work with URLs as well as local files? If so, that's really cool :)

File: tools/compare_versions/compare_versions.py (hunk 1 above)
Comment (jbkoh) [reply tools/compare_versions/compare_versions.py]: done.

**Comment by jbkoh** [comment]
> We should probably also add `semver` to the `requirements.txt` for the Brick repo

It's in this PR.

File: tools/compare_versions/README.md (hunk 2 above)
Comment (jbkoh) [reply tools/compare_versions/README.md]: Yeah `rdflib` handles both in the same way. It's a good interface.

**Comment by jbkoh** [comment]
Thanks! I will wait for travis and then merge, though there shouldn't be any problem...

**Pull request merged**